src/Symfony/Component/Security/Http/Authentication/AuthenticatorManager.php

```diff
@@ -257,4 +258,21 @@ private function handleAuthenticationFailure(AuthenticationException $authentica
         // returning null is ok, it means they want the request to continue
         return $loginFailureEvent->getResponse();
     }
+
+    private function isSensitiveException(AuthenticationException $exception): bool
+    {
+        if (!$this->hideUserNotFoundExceptions) {
+            return false;
+        }
+
+        if ($exception instanceof UserNotFoundException) {
+            return true;
+        }
+
+        if ($this->showAccountStatusExceptions) {
```

Comment:
I would expect `CustomAuthenticationMessageExceptions` to show up even if `showAccountStatusExceptions` is false, as it's the case today with `hideUserNotFoundExceptions`

Reply:
Not sure how to deal with the `CustomAuthenticationMessageExceptions` as this is no account specific exception (extends `AccountStatusException`), but a general authentication exception (extends `AuthenticationException`). 

The new key is named `hide_account_status_exceptions`, so this might be confusing.